.gitignore

```diff
@@ -7,5 +7,4 @@ env.sh
 .vscode
 *.pickle
 credentials*
-env.sh
```

Comment:
env.sh should stay in .gitignore, so that any changes made by users don't get committed, it just stays untracked.

Reply:
I removed this because there was two entries for env.sh in the .gitignore.